mixer/test/client/env/http_client.go

```diff
@@ -85,7 +85,9 @@ func HTTPPost(url string, contentType string, reqBody string) (code int, respBod
 	}
 	respBody = string(body)
 	code = resp.StatusCode
-	log.Println(respBody)
+	if len(respBody) < 10240 {
```

Comment:
let's remove it completely, I do not see the benefits of printing the response body.